di_website/spotlight/models.py

```diff
@@ -125,3 +143,44 @@ class Meta():
     ]
 
     search_fields = [index.SearchField('ddw_id'), index.SearchField('title')]
+
+
+class CountrySpotlight(TypesetBodyMixin, HeroMixin, Page):
+    spotlight_page = models.ForeignKey(
+        'wagtailcore.Page',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+',
+        help_text="Country Spotlight Page"
+    )
+    add_country_spotlight = StreamBlock([
+        ('add_spotlight_page', PageChooserBlock(required=False, target_model='spotlight.SpotlightPage')),
+    ], blank=True, help_text="Add Page")
+    country_spotlight = StreamField([
+        ('country_information', StructBlock([
+            ('title', CharBlock(required=False)),
+            ('description', RichTextBlock(
+                icon='fa-paragraph',
+                template='blocks/paragraph_block.html',
+                features=RICHTEXT_FEATURES_NO_FOOTNOTES
+            )),
+            ('spotlight_page', add_country_spotlight),
+            ('background_theme', ChoiceBlock(choices=[
+                ('light', 'Light'),
+                ('dark', 'Dark'),
+            ], help_text='Select background theme for this section')),
+        ])),
+    ], blank=True, help_text="Add Country Spotlight.")
```

Comment:
> If it's going to be this much, please add it to its own 'block' file.
Created separate block file and pushed

Reply:
> It should be one SpotlightPage for each of these ... this adds several. Create a new page, `SpotlightLocationComparisonPage` ... can only be a child of SpotlightPage. Give it a hero, a StreamField for default locations (name & geocode). Make sure these are returned by the API as well; a property `compare`, with a sub-property `default_locations`.... this is all to meet one of the requirements [here](https://github.com/devinit/data-hub-spotlights/issues/117#issuecomment-614101195).
> After that, in the template, since u have access to the spotlight_page, you should be able to acquire its location_comparison child ... can do it in the model if it makes things easier, pass it in as context.

Included `SpotlightLocationComparisonPage` as well as its data in the api and template